app/src/vanilla/java/cm/aptoide/pt/app/view/NewAppViewFragment.java

```diff
@@ -1232,13 +1242,14 @@ private SpannableString formatAppCoinsRewardMessage() {
     if (model.isDownloading()) {
       downloadInfoLayout.setVisibility(View.VISIBLE);
       install.setVisibility(View.GONE);
-      similarBottomView.setVisibility(View.GONE);
-      similarDownloadView.setVisibility(View.VISIBLE);
       showSimilarDownload = true;
+      appcInfoView.hideInfo();
```

Comment:
Probably hide/show should be done when we listen to the click on install/cancel and not when we show the download, we are mixing download ui logic with app related info.